src/Components/ArtQuiz/Views/ArtQuizWelcome.tsx

```diff
@@ -0,0 +1,40 @@
+import { Box, Flex, Text, Spacer, Button } from "@artsy/palette"
+import { OnboardingSplitLayout } from "Components/Onboarding/Components/OnboardingSplitLayout"
```

Comment:
It's probably fine to import reuse this in this fashion; it would be equally fine to just duplicate the code in here. Don't be afraid to duplicate code, especially if its under three places of use! 

(If we needed it in three or more apps we could then move it to the shared `/Components` folder, and rename)

Reply:
This PR is still WIP. I'm currently debating duplicating the code so I can place the artsy logo in the right place so the code will change here most likely.